### ISSUE TITLE
CHROME REMOTE DESKTOP NOT WORKING ANYMORE

### ISSUE THREAD

**Issue by daddyparodz** [stascorp/rdpwrap#1829]
I've installed, tested and then uninstalled successfully rdpwrapper but after uninstalling it chrome remote desktop stopped working. 
Now it says "starting" indefinitely (on the webpage https://remotedesktop.google.com/access )

What should I do to fix it? It's really bothering me because crdp was really good.
I've already tried things like uninstalling extension & software and reinstalling it, and even tried removing chrome itself but the issue remain unsolved.
(I'm opening this because I can't find ANYTHING about this particular issue).
Thanks.

**Comment by sashaqwert** [comment]
> What should I do to fix it?

This is not an RDP Wrapper issue, but a Chrome remote desktop issue.
[AnyDesk ](https://anydesk.com/)can be used as a compatible counterpart. (There is a free version for individuals)

**Comment by daddyparodz** [comment]
I don't want to use AnyDesk just because rdpwrap broke crdp. 
I just wanna know why it's broken and how to fix it because it's a rdpwrap issue the fact that it breaks other programs.

**Comment by sebaxakerhtc** [comment]
> I can't find ANYTHING about this particular issue

Because it's not an RDPWrapper issue.
You already uninstalled it, isn't it?
Check your antivirus software if it blocked/removed any system files and/or recover your OS.

**Comment by sebaxakerhtc** [comment]
Closed because of missing answer. 
Not an RDPWrapper issue

**Comment by daddyparodz** [comment]
Sorry, reopen it because the issue persists. No, I uninstalled it but it had already done the damage.
I only have windows defender and I also disabled it to try to see if it interfered, but nothing has changed.
I have already reinstalled windows without losing data, but it still does not work. The only solution would be a clean installation from scratch but I have no intention of doing it because this program caused a mess.
